### PR TITLE
add geometry.d.ts

### PR DESCRIPTION
**Pull request by nakakura**
Add Geometry Interfaces Spec

### CODE REVIEW

**Comment by vvakame** [comment]
*geometry/geometry.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/geometry
  * http://bower.io/search/?q=geometry
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (geometry/geometry-tests.ts or others)
* [ ] pass the Travic-CI test?

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,36 @@
+0 info it worked if it ends with ok
```
Comment (vvakame) [inline npm-debug.log]: please remove this file.

**Comment by vvakame** [comment]
`geometry` is already in npm. but, I do not find a good name...

**Comment by nakakura** [comment]
Thank you for your comment @vvakame san.
I changed the filename to "geometry-dom.d.ts" and removed npm-debug.log.
I searched npm and bower libraries.
There seemed to be no conflict.
I'm sorry to have troubled you

**Comment by vvakame** [comment]
@nakakura さん。thanks mate!

**Pull request merged**